src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -682,9 +682,14 @@ class TezosNodeOperator(
       val proposalsMap = proposalsState.toMap
       fetchedBlocksData.map {
         case (offset, md) =>
+          val accountIdsWithUpdatedBalances = for {
```

Comment:
It works, might it read better if extracted to a local method?

Reply:
ok, extracted